content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,176 @@
+---
+
+After you [enable the pluggable components of KubeSphere](../../../pluggable-components/), you can also uninstall them using the following commands. Please back up any necessary data before you uninstall these components.
+
+## Uninstall KubeSphere App Store
+
+```bash
+kubectl delete ns openpitrix-system
+```
+
+## Uninstall KubeSphere DevOps
+
+```bash
+helm -n kubesphere-devops-system delete ks-jenkins
+helm -n kubesphere-devops-system delete uc
+```
+
+```bash
+# Delete DevOps projects
+for devopsproject in `kubectl get devopsprojects -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch devopsprojects $devopsproject -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+for pip in `kubectl get pipeline -A -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch pipeline $pip -n `kubectl get pipeline -A | grep $pip | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+for s2ibinaries in `kubectl get s2ibinaries -A -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch s2ibinaries $s2ibinaries -n `kubectl get s2ibinaries -A | grep $s2ibinaries | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+for s2ibuilders in `kubectl get s2ibuilders -A -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch s2ibuilders $s2ibuilders -n `kubectl get s2ibuilders -A | grep $s2ibuilders | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+for s2ibuildertemplates in `kubectl get s2ibuildertemplates -A -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch s2ibuildertemplates $s2ibuildertemplates -n `kubectl get s2ibuildertemplates -A | grep $s2ibuildertemplates | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+for s2iruns in `kubectl get s2iruns -A -o jsonpath="{.items[*].metadata.name}"`
+do
+  kubectl patch s2iruns $s2iruns -n `kubectl get s2iruns -A | grep $s2iruns | awk '{print $1}'` -p '{"metadata":{"finalizers":null}}' --type=merge
+done
+
+kubectl delete devopsprojects --all 2>/dev/null
+```
+
+```bash
+kubectl delete ns kubesphere-devops-system
+```
+
+## Uninstall KubeSphere Logging
+
+To disable only log collection:
+
+```bash
+delete inputs.logging.kubesphere.io -n kubesphere-logging-system tail
+```
+
+{{< notice note >}}
+
+After running this command, KubeSphere stops collecting container logs.
+
+{{</ notice >}}
+
+To uninstall Logging system including Elasticsearch:
+
+```bash
+kubectl delete crd fluentbitconfigs.logging.kubesphere.io
+kubectl delete crd fluentbits.logging.kubesphere.io
+kubectl delete crd inputs.logging.kubesphere.io
+kubectl delete crd outputs.logging.kubesphere.io
+kubectl delete crd parsers.logging.kubesphere.io
+kubectl delete deployments.apps -n kubesphere-logging-system fluentbit-operator
+helm uninstall elasticsearch-logging --namespace kubesphere-logging-system
+```
+
+{{< notice note >}}
+
+This operation will make Auditing, Events, and Service Mesh unavailable.
```

Comment:
@wanjunlei 
Hi, I think `This operation will affect auditing, events, and Service Mesh.` is a little bit vague. So I made a small change to your original sentence. Does it look good to you?